apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/ConnectionPoolOptions.cs

```diff
@@ -59,8 +59,11 @@ public LogLevel LogLevel
                 {
                     return (LogLevel) underlyingLevel;
                 }
-                // ReSharper disable once NotResolvedInText
-                throw new ArgumentOutOfRangeException("LogLevel");
+                else
+                {
+                    // ReSharper disable once NotResolvedInText
+                    throw new ArgumentOutOfRangeException("LogLevel");
+                }
```

Comment:
It looks like this change is a no-op actually. The throw would only be reached if the if block's condition were false in both old and new versions. Same with the setter.